src/Extension.php

```diff
@@ -45,6 +46,7 @@ public function __construct($viewRenderer, $smarty)
         $smarty->registerPlugin('function', 'url', [$this, 'functionUrl']);
         $smarty->registerPlugin('function', 'set', [$this, 'functionSet']);
         $smarty->registerPlugin('function', 'meta', [$this, 'functionMeta']);
+        $smarty->registerPlugin('function', 'jsexpr', [$this, 'functionJsExpr']);
```

Comment:
How about naming it just `js`?

Reply:
i really dont know. You can decide it.

Suggestions: 
- `js`
- `javascript`
- `jsexpr`

Reply:
`js` is short and still pretty much understandable.